srcpkgs/ksuperkey/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'ksuperkey'
+pkgname=ksuperkey
+version=0.4
+revision=1
+build_style=gnu-makefile
+make_use_env=compliant
```

Comment:
Interesting.
Out of curiosity, what `make_use_env=compliant` is supposed to do?

Reply:
I don't see `make_use_env=compliant` in the Manual.